src/containers/guest-home-page/signup-form/SignupForm.jsx

```diff
@@ -143,7 +143,13 @@ const SignupForm = ({
       </Box>
 
       <AppButton
-        disabled={!isAgreementChecked}
+        disabled={
+          !isAgreementChecked ||
+          !data.firstName ||
+          !data.lastName ||
+          t(errors.email) ||
+          t(errors.confirmPassword)
```

Comment:
Why we need to call function `t` when we check only existing of errors, not its text?
```suggestion
          errors.email ||
          errors.confirmPassword
```

Reply:
You were right , i removed t() function and code still works . Also I have found one more validation bug, so I fixed it. I’m going to merge my branch